spec/features/recipes/recipes_page_spec.rb

```diff
@@ -0,0 +1,11 @@
+feature 'user visits recipes page' do
+  before do
+    FactoryBot.create(:recipe, :scrambled_eggs)
+    visit('/')
+  end
+
+  it 'shows user an example data' do
+    expect(page).to have_content Recipe.first.name
+    expect(page).to have_content Recipe.first.instructions
```

Comment:
instructions shouldn't be visible on the `index` page. In some cases it can be very large test which will make it look bad. We will add it later to `show`.